internal/gps/prune.go

```diff
@@ -107,152 +111,141 @@ func pruneNestedVendorDirs(baseDir string) error {
 	return filepath.Walk(baseDir, stripVendor)
 }
 
-// pruneUnusedPackages deletes unimported packages found within baseDir.
+// pruneVendorDirs deletes all nested vendor directories within baseDir.
```

Comment:
why keep the commented funcs?

Reply:
That's why this is a [WIP]. 😁 

I trying to refactor strip vendor `func`s to use the `filesystemState`.